examples/contrib/readhar.py

```diff
@@ -0,0 +1,141 @@
+"""Reads HAR files into flow objects"""
+import asyncio
+import json
+import logging
+import time
+
+from mitmproxy import command
+from mitmproxy import connection
+from mitmproxy import ctx
+from mitmproxy import exceptions
+from mitmproxy import http
+from mitmproxy import types
+
+logger = logging.getLogger(__name__)
+
+
+class ReadHar:
+    def __init__(self):
+        self.flows = []
+
+    def fix_headers(
+        self, request_headers: list[dict[str, str]] | list[tuple[str, str]]
+    ) -> list[tuple[bytes, bytes]]:
+        """Converts provided headers into (b"header-name", b"header-value") tuples"""
+        flow_headers = []
+        for header in request_headers:
+            # Applications that use the {"name":item,"value":item} notation are Brave, Chrome, Edge, Firefox, Charles, Fiddler, Insomnia, Safari
+            if isinstance(header, dict):
+                try:
+                    key = header["name"]
+                    value = header["value"]
+                except KeyError as e:
+                    raise exceptions.OptionsError(str(e)) from e
```

Comment:
Why are we raising OptionsError here? Given that the input is malformed, we should a) raise an appropriate error type (ValueError makes most sense I think), or b) not do any error handling and let the KeyError propagate as-is.